packages/jest-reporters/src/utils.ts

```diff
@@ -139,11 +169,27 @@ export const getSummary = (
 
   const tests =
     chalk.bold('Tests:       ') +
-    (testsFailed ? chalk.bold.red(`${testsFailed} failed`) + ', ' : '') +
-    (testsPending ? chalk.bold.yellow(`${testsPending} skipped`) + ', ' : '') +
-    (testsTodo ? chalk.bold.magenta(`${testsTodo} todo`) + ', ' : '') +
-    (testsPassed ? chalk.bold.green(`${testsPassed} passed`) + ', ' : '') +
-    `${testsTotal} total`;
+    (testsFailed
```

Comment:
woah, hard to read. thoughts on a `.map().filter().join()` instead? might not help, dunno. if not, maybe a helper of some sort?